resources/dw/resource_dw_acc_test.go

```diff
@@ -166,7 +166,8 @@ func TestAccDwCluster_Basic(t *testing.T) {
 					testAccAwsDataLakeConfig(&dlParams),
 					testAccAwsClusterBasicConfig(&envParams),
 					testAccDwCatalog(),
-					testAccHiveVirtualWarehouse(cdpacctest.RandomShortWithPrefix("tf-hive"))),
+					testAccHiveVirtualWarehouse(cdpacctest.RandomShortWithPrefix("tf-hive")),
```

Comment:
I think you are missing a )

Reply:
The line with 3 brackets is the one below this line.

Reply:
oh, makes sense